etc/cms-resource-reporter.yml

```diff
@@ -0,0 +1,120 @@
+# etc/cms-resource-reporter.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Handles CMS Resource SNS Events
+Parameters:
+  SlackMessageRelayTopicArn:
+    Type: String
+  CmsSeriesCreateSnsTopicArn:
```

Comment:
Can this stack be generalized to monitor any announce arn?  Maybe using the SNS topic parameter (create/update/delete), plus a few labels/nouns to describe it ("was created" / "was updated").  Or am I over thinking this.

Reply:
Took a shot at an abstract resource reporter in 7a1990b.  It ends up being a less nicely formatted, but I think still very usable/practical.